zoo/policies/chase_via_points_agent.py

```diff
@@ -7,11 +7,14 @@
 
 class ChaseViaPointsAgent(Agent):
     def act(self, obs: Observation):
-        assert obs.waypoint_paths, (
+        assert obs.waypoint_paths is not None, (
             f"Waypoint paths = {obs.waypoint_paths}; "
             "cannot be empty or None. Enable waypoint paths in agent interface."
         )
 
+        if len(obs.waypoint_paths) == 0:
+            return (1, 0)
```

Comment:
This is likely to let unexpected cases to slip through unnoticed, leaving us with lesser chances of detecting and fixing any issues.